src/java.base/share/classes/java/nio/channels/Channels.java

```diff
@@ -342,7 +343,8 @@ protected void implCloseChannel() throws IOException {
      *
      * <p> The resulting channel will not be buffered; it will simply redirect
      * its I/O operations to the given stream.  Closing the channel will in
-     * turn cause the stream to be closed.  </p>
+     * turn cause the stream to be closed.  The returned channel implements
+     * {@linkplain Flushable}; flushing the channel will flush the stream. </p>
```

Comment:
This was just a suggestion to try but I think it's a bit too weird so I would drop that. 

I think the main thing we need to decide is whether a factory method for an auto-flushing output stream is useful to have to not. Maybe we should try to survey the usages of this factory method to see if there is much usage and in what contexts.